patches/components-sync-protocol-device_info_specifics.proto.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/components/sync/protocol/device_info_specifics.proto b/components/sync/protocol/device_info_specifics.proto
-index e4cea525835abc105feee7480106f3348d10d534..ce1f276c7f7ee6d53a7abfa91fd01efc6348efff 100644
+index da74199a29a033e9159f2b30293b3f86778db7f2..8def3120dc6ec75804d5f28dbd39e50bb080ad75 100644
```

Comment:
Should this PR include these patch files update??

Reply:
When I rebased the https://github.com/brave/brave-core/pull/6732 after master was upgraded to C87, some patches were required to be fixed, I committed these changes. The patches which had just only modified indexes I didn't committed, as they were applied without errors. This was mistake, because run of `npm run update_patches` recreate them and they were different. In this PR I included those changes, but then happened `Update from Chromium 87.0.4280.27 to Chromium 87.0.4280.40` and the patches and indexes were fixed, so my commit was dismissed on new rebase.

So the answer is no. And now that commit is removed.